ci/script.sh

```diff
@@ -28,6 +28,7 @@ if [ "$TRAVIS_OS_NAME" == "windows" ]; then
         cp "./target/debug/winpty-agent.exe" "./target/debug/deps"
     fi
     cargo test -p winpty || error=true
+    ./wix/build.sh
```

Comment:
Builds should never be done for normal CI builds, but only before deployment.

Reply:
Since you've forked Alacritty you can test deployment by creating releases for your local branch. That actually tests properly if the releases will be uploaded and everything.

Reply:
Oh okay, thanks I'll try that.